modules/table/src/main/java/org/apache/ignite/internal/table/distributed/raft/PartitionListener.java

```diff
@@ -111,17 +116,7 @@ public void onRead(Iterator<CommandClosure<ReadCommand>> iterator) {
         iterator.forEachRemaining((CommandClosure<? extends ReadCommand> clo) -> {
             Command command = clo.command();
 
-            if (!tryEnlistIntoTransaction(command, clo)) {
-                return;
-            }
-
-            if (command instanceof GetCommand) {
-                clo.result(handleGetCommand((GetCommand) command));
-            } else if (command instanceof GetAllCommand) {
-                clo.result(handleGetAllCommand((GetAllCommand) command));
-            } else {
-                assert false : "Command was not found [cmd=" + clo.command() + ']';
-            }
+            assert false : "No read commands expected, [cmd=" + clo.command() + ']';
```

Comment:
`assert false`? There must be some explanation of such code

Reply:
There's such explanation: No read commands expected